recipes/pyopengl/post-link.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# Test for OpenGL Driver on Linux
+if [ `uname` == Linux ]; then
+    $PYTHON -c "import OpenGL.GL"
```

Comment:
Hmm...interesting. So, it looks like they will tell you the [`PREFIX`]( http://conda.pydata.org/docs/building/build-scripts.html ). Maybe we could hack something in that uses that if nothing else works?